src/models/IStop.ts

```diff
@@ -2,7 +2,7 @@ export default interface IStop {
     nodeId: string;
     municipality: string;
     nameFi: string;
-    nameSe: string;
+    nameSe: string; // TODO: rename as nameSw
```

Comment:
Standard swedish should be Sv, https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes

Reply:
Hmm, then our routePathProperties should be renamed too.. they end with xxxSw

Reply:
Hmm, I have no difference which way we go, both Se, Sw and Sv goes well for me. But it would probably be good to be consistent. Personally I think Sw sounds most describable, but Sv is the standard. Which way should we go?

Reply:
I would go with Sw which is closer to "Swedish", I don't like "nameSv" that much

Reply:
okay, lets go with Sw